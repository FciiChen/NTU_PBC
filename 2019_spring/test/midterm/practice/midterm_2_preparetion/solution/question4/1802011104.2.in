從十月中成案以來，東京奧運正名公投案就因為背負著國人對於「正名」的期待，以及可能對於我國運動員權益造成重大影響，成為引發最多討論的一個公投案，「維護運動員參賽權益」、「中華台北不是個國家」兩派劍拔弩張，也使得今天公投結果牽動我國體壇的未來走向。

東奧正名公投案的代表包括紀政、前體委會主委楊忠和、李登輝民主協會理事長張燦鍙、民報文化事業創辦人陳永興等，從連署階段就引發國際奧會的關注，包括五月來信強調不會允准我國名稱變動，近兩個月更兩度來信關切公投程序，引發是否可能導致中華奧會遭停權或中止會籍的討論。

但在正方再三為我國選手參賽權掛保證，甚至提出了只有中華台北奧會向國際奧會自告洋狀，才會導致我國遭停權的狀況，讓民眾對於東奧正名公投是霧裡看花，外界更有假使正名公投通過。不論中華台北奧會是否提出改名申請，最壞狀況就是我國選手仍以中華台北參賽的說法，讓不了解國際奧會及國際體壇的民眾有了同意的理由。

事實上在國際奧會來函中提到，任何試圖迫使中華台北奧會破壞洛桑協議，或是違反國際奧會執委會決定的做法，都被視為外部干涉。體育界人士也認為，假使東奧正名公投最終通過，就可能被國際奧會視為違反五月在洛桑舉行的會議上「不允准」中華台北奧會名稱變更的決定，讓我國面臨停權風險。

東奧正名公投的影響力，也不僅是在東京奧運參賽權上，如中華奧會最終是遭到國際奧會停權或撤銷會籍，也代表我國在國際單項協會的會籍遭取消，等同於世界盃、世錦賽、亞錦賽、亞洲盃等賽事都將無權參加；僅有不用透過單項協會報名的職業聯賽不受影響，如高球ＬＰＧＡ與網球ＡＴＰ、ＷＴＡ等。

至於外界質疑東奧正名公投是假議題，即使投同意也不影響我國參賽名稱與運動員參賽權益，但在國際奧會握有主導權的情況下，都不該忽視此公投結果所可能導致的後果，這也是國內體壇最憂心的地方，因為任何影響運動員參賽權益的風險都不該嘗試。
BREAK