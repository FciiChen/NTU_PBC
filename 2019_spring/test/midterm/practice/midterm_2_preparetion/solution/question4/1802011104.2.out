國_奧 8
際_會 8
名_投 7
正_公 7
中_台 6
奧_名 6
華_北 6
參_權 5
東_正 5
北_會 4
